src/mesh/index_derivs.cxx

```diff
@@ -219,7 +219,7 @@ BoutReal FDDX_U1(stencil &v, stencil &f) {
   vs = 0.5*(v.c + v.p);
   result -= (vs >= 0.0) ? vs * f.c : vs * f.p;
 
-  return result;
+  return - result;
```

Comment:
Looks reasonable to me, but I could be wrong.

Should we also change the staggered version (https://github.com/boutproject/BOUT-dev/blob/41d05ad89252993a271aa411265bf31857a77e90/src/mesh/index_derivs.cxx#L386)?

Reply:
@d7919 `FDDX_U1_stag` should be fixed now.